superset-frontend/src/SqlLab/components/TableElement.tsx

```diff
@@ -92,15 +114,15 @@ const TableElement = props => {
           />
         );
       }
-      let latest = Object.entries(table.partitions?.latest || []).map(
+      const latest = Object.entries(table.partitions?.latest || []).map(
```

Comment:
small nit, but if we're not going to use latest again, can we just define it once as 
```
const latest = Object.entries(table.partitions?.latest || []).map(
    ([key, value]) => `${key}=${value}`,
).join('/');
```